src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderConfig.tsx

```diff
@@ -373,38 +373,34 @@ export const LimitOrderConfig = ({
   }, [marketPriceCryptoPrecision, priceAsset.symbol])
 
   return (
-    <Stack spacing={4} px={6} py={4}>
+    <Stack spacing={6} px={6} py={4}>
       <Flex justifyContent='space-between' alignItems='center'>
         <HStack>
           <Text translation='limitOrder.when' />
-          <Button
-            variant='unstyled'
+          <RawText
```

Comment:
q: why are we replacing the button to a Text?

It's less accessible, Texts are not focusable by keyboards by default, screenreaders might not announce it as a button, it's less semantic...

Reply:
The button was adding a lot of extra padding, so I matched the rate setup. I can try and update the button accordingly if that is the case about accessibility

Reply:
> The button was adding a lot of extra padding, so I matched the rate setup. I can try and update the button accordingly if that is the case about accessibility

I think it would be better to keep the button and override paddings then! 🙏 

Even thought web3 is not so accessible in general, small efforts can be done

Reply:
Cool, thanks for thinking of the accessibility and informing me as such. Update incoming

Reply:
e190fdcca65c74ae898374b7e5284e3c2a5d3713